docs/reference/commandline/cli.md

```diff
@@ -48,6 +48,10 @@ by the `docker` command line:
 * `DOCKER_TLS_VERIFY` When set Docker uses TLS and verifies the remote.
 * `DOCKER_CONTENT_TRUST` When set Docker uses notary to sign and verify images.
   Equates to `--disable-content-trust=false` for build, create, pull, push, run.
+* `DOCKER_CONTENT_TRUST_SERVER` The URL of the Notary server to use. This defaults
+  to the same URL as the registry.
+  Note: This environment variable is likely to be deprecated in favor of a better
```

Comment:
I get the idea, but I'm not sure we should notify people that something *may* be replaced. In general we don't give these heads up, until there's a deprecation path in place